src/script/view_model/bindings/MessageListBindings.js

```diff
@@ -83,9 +83,7 @@ ko.bindingHandlers.infinite_scroll = {
     const {onHitTop, onHitBottom, onInit} = params();
     onInit(scrollingElement);
 
-    const onScroll = event => {
-      const element = event.target;
-
+    const onScroll = ({target: element}) => {
```

Comment:
Is it an `event` or an `element` that comes in? Because before it seemed to be `event.target`.